style.css

```diff
@@ -438,3 +439,188 @@ textarea {
   justify-content: left;
   cursor: pointer;
 }
+
+#navtag {
+  display: none;
+}
+
+/* Styling for desktop version of project */
+
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    box-sizing: 0;
+  }
+
+  #wrapper {
+    width: 100%;
+    background-size: contain;
+    margin: 0;
+  }
+
+  #navbar {
+    display: none;
+  }
+
+  #navtag {
+    display: block;
+  }
+
+  #container {
+    display: flex;
+    column-gap: 950px;
+    width: 100%;
+    margin: 0;
+  }
+
+  .navigation {
+    display: flex;
+    flex-direction: row;
+    justify-content: right;
+    column-gap: 30px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .logo {
+    margin-left: 55px;
+  }
+
+  #name {
+    width: 700px;
+  }
+
+  #section {
+    margin-left: 260px;
+    margin-bottom: 255px;
+  }
+
+  #about {
+    width: 920px;
+  }
+
+  .content {
+    display: flex;
+    padding-left: -50px;
+    justify-content: left;
+    height: 496px;
+    margin-left: 142px;
+    margin-right: 142px;
+    column-gap: 46px;
+  }
+
+  div.content > img {
+    width: 544px;
+    height: 448px;
+  }
+
+  .reverse {
+    flex-direction: row-reverse;
+  }
+
+  .row {
+    display: inline-grid;
+    grid-template-columns: auto auto auto auto auto;
+    column-gap: 7px;
+    margin-left: -38px;
+    margin-top: -25px;
+  }
+
+  .cardcontent {
+    margin-left: 24px;
+  }
+
+  .final {
+    display: flex;
+    flex-direction: row;
+  }
+
+  #about_section {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    padding-bottom: 100px;
+  }
+
+  .abouttwo {
+    padding-left: 500px;
+    margin-top: -420px;
+    margin-right: 0;
+  }
+
+  .aboutone {
+    padding-left: 142px;
+  }
+
+  .arrow {
+    width: 80%;
+  }
+
+  .last {
+    width: 447px;
+  }
+
+  .stacked {
+    display: flex;
+    flex-direction: column;
+    width: 122px;
+    height: 120px;
+    margin-right: 0;
+  }
+
+  .listing {
+    padding-left: 0;
+  }
+
+  .stack_list {
+    display: flex;
+  }
+
+  #framework_section {
+    width: 80%;
+    margin-right: 0;
+  }
+
+  #form-wrap {
+    width: 100%;
+  }
+
+  .form-text {
+    width: 569px;
+    text-align: center;
+  }
+
+  #form-section {
+    display: grid;
+    justify-content: center;
+  }
+
+  form {
+    width: 100%;
```

Comment:
- [ ] I think it would be better to replicate the ```input``` height and width and match it with that of [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848&t=lm66x04u5euYTBoh-0)
   -  You may kindly follow the following properties :point_down: 
   
    ![image](https://user-images.githubusercontent.com/102757126/206629515-06808a4f-f67f-46d7-bb10-de248fcf00ae.png)

- [ ] It would be better also to use the same background as that of the Figma design. Remember it is important that you try to replicate the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848&t=lm66x04u5euYTBoh-0) as closely as possible to give confidence to recruiters in your ability to produce exact business requirements :pray:

**Your design**
![image](https://user-images.githubusercontent.com/102757126/206630319-29d96b90-55f2-4229-a7ef-4d7263112ce4.png)
**Figma design**
![image](https://user-images.githubusercontent.com/102757126/206630399-2feb9388-807d-40b2-88f8-27018179f122.png)

Reply:
- [ ] While doing a great implementation which the previous code reviewer suggested, you introduce new bugs in the entire design of the project. Kindly fix :wrench: the issue in the screenshot below and compare it with the design template you choose.

***Your design in desktop version***
>![Screenshot from 2022-12-09 13-08-53](https://user-images.githubusercontent.com/65129228/206685238-98146d93-d680-4d6f-a57e-f574028bce89.png)
![Screenshot from 2022-12-09 13-10-26](https://user-images.githubusercontent.com/65129228/206685347-461429ca-76ac-44c7-a5af-c5f88a4041fe.png)
![Screenshot from 2022-12-09 13-10-53](https://user-images.githubusercontent.com/65129228/206685426-d2d9ecc5-db58-48fb-a64b-dafc45945e2c.png)